docs/TextRegion.md

```diff
@@ -0,0 +1,86 @@
+_New in [:octicons-tag-24: 2.7.7](https://github.com/py-pdf/fpdf2/blob/master/CHANGELOG.md)_
```

Comment:
This documentation currently does not make any reference to `TextColumns.md`

I think we should try to create links between those those 2 pages are they very closely related,
and reading one can help users to better understand the other.

Reply:
There should indeed be some cross linking. Looks like I forgot that when splitting the two pages.